src/node/desktop/src/main/preferences/electron-desktop-options.ts

```diff
@@ -349,3 +351,43 @@ if (process.platform === 'darwin') {
 } else {
   defaultFonts = ['Ubuntu Mono', 'Droid Sans Mono', 'DejaVu Sans Mono', 'Monospace'];
 }
+
```

Comment:
There are missing changes in this source file. See https://github.com/rstudio/rstudio/pull/12970.